tests/e2e/vmgroups_test.go

```diff
@@ -111,6 +111,9 @@ func (vg *VmGroupTest) TearDownSuite(c *C) {
 	out, err = ssh.InvokeCommand(vg.config.EsxHost, cmd)
 	log.Printf(out)
 
+	// Remove Config DB
+	adminutils.ConfigRemove(vg.config.EsxHost)
+
```

Comment:
If we decide to do this, we must make sure each test clean up itself properly. For example, if a test create a VM group, it must remove the VM group when test end (not only in successful case, but also in failure case).  I don't think current test code did this.